CMakeLists.txt

```diff
@@ -3920,13 +3920,15 @@ endif (gRPC_BUILD_CODEGEN)
 
 if (gRPC_BUILD_CODEGEN)
```

Comment:
Was this already fixed by other PR?

It looks like the `if (gRPC_BUILD_CODEGEN)` guards are already there :-)

Reply:
https://github.com/grpc/grpc/pull/18151/files